### PR TITLE
feat: add Laravel 11.x compatibility

### PR DESCRIPTION
**Pull request by mjblacker**
Upgrades to dependencies for Laravel 11.x

Main upgrade is to the tests which needed another library to replace testing the feature withConsecutive from PHPUnit that was removed in version 10.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
     "require": {
         "php": "^8.1",
-        "illuminate/console": "^9.0|^10.0",
-        "illuminate/support": "^9.0|^10.0"
+        "illuminate/console": "^9.0|^10.0|^11.0",
+        "illuminate/support": "^9.0|^10.0|^11.0"
     },
     "require-dev": {
-        "pestphp/pest": "^1.22.3"
+        "pestphp/pest": "^1.22.3|^2.34"
```
Comment (owenvoke) [inline composer.json]: Feel free to drop support for Pest 1.x, and Laravel 9.x 👍🏻

File: composer.json [hunk 2]
```diff
@@ -15,14 +15,15 @@
...
     },
     "autoload": {
         "psr-4": {
+            "NunoMaduro\\Tests\\LaravelConsoleTask\\": "tests/",
```
Comment (owenvoke) [inline composer.json]: Please can this be added to `autoload-dev` instead? 👀

**Review by owenvoke — CHANGES_REQUESTED** [review]
Thank you for your effort on this, I appreciate it! Just a couple of comments.

File: composer.json (hunk 2 above)
Comment (mjblacker) [reply composer.json]: Yep

File: composer.json (hunk 1 above)
Comment (mjblacker) [reply composer.json]: Nice, removed!

**Pull request merged**